Dockerfile

```diff
@@ -13,18 +13,16 @@ COPY routes /envelope-game/routes
 COPY __tests__ /envelope-game/__tests__
 COPY __mocks__ /envelope-game/__mocks__
 
-RUN yarn test
+RUN npm test
 
-RUN yarn run build
+RUN npm run build
 
-RUN yarn install --production --ignore-scripts --prefer-offline
+# RUN npm prune --production
```

Comment:
if we don't need to do this, we should remove this line.

Reply:
agreed, removed